renderer/src/actions/editingActions.ts

```diff
@@ -0,0 +1,19 @@
+export type EditingHistoryAction<T> =
```

Comment:
Choose one way - do it as a string everywhere or make it use generics everywhere. I see no reason for making the code more complex and not even using advantages that you receive by making it like this.

Reply:
Resolved